api/applications/serializers.py

```diff
@@ -8,31 +8,34 @@
 
 class ApiOAuth2ApplicationSerializer(JSONAPISerializer):
     """Serialize data about a registered OAuth2 application"""
-    client_id = ser.CharField(help_text="The client ID for this application (automatically generated)",
+    id = ser.CharField(help_text='The client ID for this application (automatically generated)',
+                       read_only=True,
```

Comment:
The `label` parameter also affects the form labels in a the browsable API:

![django_rest_framework](https://cloud.githubusercontent.com/assets/2379650/9691848/21257970-5313-11e5-87ac-5bf9708271a5.png)